package.json

```diff
@@ -91,29 +87,28 @@
     ]
   },
   "devDependencies": {
-    "@babel/cli": "~7.12.1",
-    "@babel/core": "~7.12.1",
-    "@babel/preset-env": "~7.12.1",
-    "@babel/preset-react": "~7.12.1",
-    "@babel/register": "~7.12.1",
-    "@material-ui/core": "^4.8.0",
-    "@material-ui/icons": "^4.0.0",
-    "babel-loader": "~8.1.0",
-    "css-loader": "~2.1.1",
-    "eslint-plugin-react-hooks": "^4.1.2",
-    "jest": "~26.6.2",
-    "jest-pnp-resolver": "~1.2.2",
-    "jest-resolve": "~26.6.2",
-    "js-cookie": "~2.2.1",
-    "mini-css-extract-plugin": "0.5.0",
-    "node-sass": "~4.14.1",
-    "react-test-renderer": "16.13.1",
-    "standard": "^14.3.0",
-    "style-loader": "~0.23.1",
-    "typeface-roboto": "0.0.54",
-    "url-loader": "1.1.2",
-    "webpack": "4.44.x",
-    "webpack-bundle-tracker": "^1.2.0",
-    "webpack-cli": "4.3.x"
+    "@babel/cli": "~7.17.6",
+    "@babel/core": "~7.17.8",
+    "@babel/preset-env": "~7.16.11",
+    "@babel/preset-react": "~7.16.7",
+    "@babel/register": "~7.17.7",
+    "@material-ui/core": "^4.12.3",
+    "@material-ui/icons": "^4.11.2",
+    "babel-loader": "~8.2.4",
+    "css-loader": "~6.7.1",
+    "eslint-plugin-react-hooks": "^4.4.0",
+    "jest": "~27.5.1",
+    "jest-resolve": "~27.5.1",
+    "js-cookie": "~3.0.1",
+    "mini-css-extract-plugin": "2.6.0",
+    "react-app-polyfill": "^3.0.0",
+    "react-test-renderer": "17.0.2",
+    "standard": "^16.0.4",
+    "style-loader": "~3.3.1",
+    "typeface-roboto": "1.1.13",
+    "url-loader": "4.1.1",
+    "webpack": "5.70.x",
+    "webpack-bundle-tracker": "^1.4.0",
+    "webpack-cli": "4.9.x"
```

Comment:
It looks like we could maybe optimize how `node_modules` is built. Some of the dependencies under `devDependencies` are more like `dependencies` (`@material-ui`, `js-cookie`, `typeface-roboto`?). Looking at the Docker artifacts, I see there is an effort to remove extraneous packages (`npm prune --production`), but I'm not sure if it's actually working. When I go to OpenShift and peak inside `node_modules`, I still see `jest`, `webpack`, `-loader`, etc. We should probably also make sure that the frontend unit test and related artifacts are left behind from the production images (currently, they are under `assets/__tests__`).

Reply:
I'd like to discuss with @jonespm but this is probably best tackled separately at later date.

Reply:
It looks like @justin0022 originally added the `npm prune` afd74b5a8c654d6c0100b427210ba2751bc97f8a.

I've never checked to see if it's working either. 

It looks like it might have to be combined with [node-prune ](https://github.com/tj/node-prune) to really remove the node_modules. I saw that mentioned in this article here: https://itsopensource.com/how-to-reduce-node-docker-image-size-by-ten-times/

I think we can just address this in a separate issue but yeah having the appropriate things in `devDependencies` makes sense to me.